apple/swift-docc
# Mark automatic curation topic sections as "generated" in Render JSON

Bug/issue #, if applicable: rdar://116584446

## Summary

This fixes a tiny inconsistency in the Render JSON data where the generated topic sections for auto-curated symbols aren't marked as "generated" but the topic sections for auto-curated articles are.

## Dependencies

None

## Testing

- Build documentation for a small project with a mix of manual and automatic curation. 
- Inspect the `topicSections` in the Render JSON:
  Topic sections for auto-curated symbols should be marked as "generated". For example:
  ```
  "topicSections": [
    {
      "title": "Variables",
      "identifiers": [
        "doc://SomeModule/documentation/SomeStruct/firstVariable",
        "doc://SomeModule/documentation/SomeStruct/secondVariable"
      ],
      "generated": true
    },
    ...
  ```

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
